5.5/s2i/bin/run

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# Export the Cgroup limits
+export $(/usr/bin/cgroup-limits.py)
```

Comment:
bad, this will ignore errors and behave oddly if the script throws an exception.

Btw, should this run script use set -e?